FzzBuzz.py

```diff
@@ -0,0 +1,12 @@
+# Напишите программу, которая печатает цифры от 1 до 100,
+# но вместо чисел, кратных 3 пишет Fizz, вместо чисел кратных 5 пишет
+# Buzz, а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+for i in range(1, 101):
+    if i % 3 == 0 and i % 5 == 0:

```

Comment:
15?

Reply:
15 multiple of 3 and 5. Output: FizzBuzz

Reply:
oh, sry, I understood, rdy